src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -918,6 +918,25 @@ executing the TerminateProcess method of the process. Whereas the SIGKILL is equ
 the forceful removal of the running process.
 
 
+===== Termination Behavior by Language
+
+.Termination Behavior by Language
+|===
+|Language/Execution type|Bash |Python
```

Comment:
Yes good point. But I didn't test. Because each language behaves different. E.g. in Java the non-forked mode: probably an InterruptedException is throw. In forked mode probably nothing and can only be handled with a shutdownhook.